nym-vpn-core/nym-vpn-lib/src/routing.rs

```diff
@@ -246,34 +269,7 @@ pub async fn setup_mixnet_routing(
 ) -> Result<tun2::AsyncDevice> {
     debug!("Creating tun device");
     let mixnet_tun_config = config.mixnet_tun_config.clone();
-    #[cfg(target_os = "android")]
-    let mixnet_tun_config = {
-        let mut tun_config = talpid_tunnel::tun_provider::TunConfig::default();
-        let tun_ips = config.tun_ips();
-        tun_config.addresses = vec![tun_ips.ipv4.into(), tun_ips.ipv6.into()];
-        let fd = {
-            let mut tun_provider = config
-                .tun_provider
-                .lock()
-                .expect("access should not be passed to mullvad yet");
-
-            let fd = tun_provider.get_tun(tun_config)?.as_raw_fd();
-            if let Some(raw_fd) = config.gateway_ws_fd {
-                tun_provider.bypass(raw_fd)?;
-            }
-            fd
-        };
-        let mut mixnet_tun_config = mixnet_tun_config.clone();
-        mixnet_tun_config.raw_fd(fd);
-        mixnet_tun_config
-    };
-    #[cfg(target_os = "ios")]
-    let mixnet_tun_config = {
```

Comment:
Ouch you have removed the mixnet tunnel configuration for iOS

Reply:
I think this should be fine. We have a separate scoped method for setting up mixnet for mobile.